tests/base/test_status.py

```diff
@@ -5,36 +5,35 @@
 
 class TestStatus():
     def test_hello(self):
-        i =1 
+        i =1
         assert 1 == i
 
     def test_status_codes(self):
         status_code_available = [
-            "StatusCode.kOk", 
-            "StatusCode.kCancelled", 
-            "StatusCode.kUnknown", 
+            "StatusCode.kOk",
+            "StatusCode.kCancelled",
+            "StatusCode.kUnknown",
             "StatusCode.kInvalidArgument",
-            "StatusCode.kDeadlineExceeded", 
-            "StatusCode.kNotFound", 
-            "StatusCode.kAlreadyExists", 
-            "StatusCode.kPermissionDenied", 
-            "StatusCode.kResourceExhausted", 
+            "StatusCode.kDeadlineExceeded",
+            "StatusCode.kNotFound",
+            "StatusCode.kAlreadyExists",
+            "StatusCode.kPermissionDenied",
+            "StatusCode.kResourceExhausted",
             "StatusCode.kFailedPrecondition",
-            "StatusCode.kAborted", 
+            "StatusCode.kAborted",
             "StatusCode.kOutOfRange",
             "StatusCode.kUnimplemented",
-            "StatusCode.kInternal", 
-            "StatusCode.kUnavailable", 
-            "StatusCode.kDataLoss", 
-            "StatusCode.kUnauthenticated", 
+            "StatusCode.kInternal",
+            "StatusCode.kUnavailable",
+            "StatusCode.kDataLoss",
+            "StatusCode.kUnauthenticated",
         ]
         actual_codes = []
-        x = range(17)
-        for n in x:
+        for n in range(17):
```

Comment:
You can replace the For Loop by a List comprehension if you really want to get the code shorter
`actual code = [str(pd.Status.StatusCode(n)) for n in range(17)]`
But I guess here since it is a test example we care about readability